src/modm/platform/iwdg/stm32/iwdg.hpp

```diff
@@ -46,6 +46,11 @@ class Iwdg : public ::modm::PeripheralDriver
 	};
 
 public:
+	static inline void
+	initialize(Prescaler prescaler, uint16_t reload)
+	{
+		configure(prescaler, reload);
+	}
```

Comment:
Let's instead rename `configure` to `initialize` and move it here to avoid the unnecessary forwarding.

Reply:
Just pushed a version without the forwarding. Please also don't forget the PR #1136.